src/rpm/headers/header.rs

```diff
@@ -365,7 +373,7 @@ impl Header<IndexSignatureTag> {
     ) -> Result<(), RPMError> {
         self.write_async(out).await?;
         // align to 8 bytes
-        let padding_needed = 8 - (self.index_header.data_section_size % 8);
+        let padding_needed = (8 - (self.index_header.data_section_size % 8)) % 8;
```

Comment:
nit: Maybe worth a tiny `const fn padding_needed(len: usize) -> usize {..}`

Reply:
`data_section_size` isn't known at compile time so I don't think it would ever be executed in `const` context